commons/src/main/java/org/infinispan/commons/dataconversion/EncodingException.java

```diff
@@ -0,0 +1,14 @@
+package org.infinispan.commons.dataconversion;
+
+/**
+ * @since 9.1
+ */
+public class EncodingException extends RuntimeException {
```

Comment:
I don't see this exception being used much. Current encoders seem to prefer throwing CacheException.